estate/models/estate_property.py

```diff
@@ -0,0 +1,21 @@
+from odoo import fields, models
+
+
+class Property(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate application"
+
+    name = fields.Char('Name', required=True)
+    description = fields.Text('Description')
+    postcode = fields.Float('Postcode')
+    date_availability = fields.Date('Date availability')
+    expected_price = fields.Float('Expected price', required=True)
+    selling_price = fields.Float('Selling price')
+    bedrooms = fields.Integer('Bedrooms')
+    living_area = fields.Integer('Living area')
+    facades = fields.Integer('Facades')
+    garage = fields.Boolean('Garage')
+    garden = fields.Boolean('Garden')
+    garden_area = fields.Integer('Garden area')
+    garden_orientation = fields.Selection(string='Garden Orientation',
+                                          selection=[('north', 'North'), ('south', 'South'), ('est', 'Est'), ('west', 'West')])
```

Comment:
I prefer that way to be more readable for the selection fields
```suggestion
    garden_orientation = fields.Selection([
        ('north', 'North'),
        ('south', 'South'),
        ('east', 'East'),
        ('west', 'West')
    ], string='Garden Orientation',)
```

Reply:
Done! Thanks